prow/config.yaml

```diff
@@ -3753,3 +3753,696 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+
+- interval: 2h
+  name: ci-kubernetes-e2e-gce-gci-qa-master
```

Comment:
The job used to run every 5min, and the interval count towards start time, so it would not make that much difference.. How about I let recent one runs more often, and turn down frequency for the old ones?